File: 1-Iniciante/ex1173.c
#include <stdio.h>

int main(){
    /*variaveis*/
    int n[10], i, x;

    /*entrada*/
    scanf("%d", &x);

    /*processamento*/
    n[0] = x;
    for(i = 1; i < 10; i++){
        n[i] = n[i - 1] * 2;
    }

    /*saida*/
    for(i = 0; i < 10; i++){
        printf("N[%d] = %d\n", i, n[i]);
    }
    return 0;
}

File: 1-Iniciante/ex1173.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8'); // adicione uma barra "/" antes do "dev" para ser aceito no beecrowd
let valor = parseInt(input);

// processamento & saida
let vetor = [valor];
console.log(`N[0] = ${valor}`);
for(let i = 1; i < 10; i++){
    vetor.push(vetor[i - 1] * 2);
    console.log(`N[${i}] = ${vetor[i]}`);
}